task3/src/main/java/Player.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Player class, contains the player's name.
+ *
+ * @author Kamil Best
+ */
+class Player {
+    private String playerName;
+
+    Player(String name) {
+        playerName = name;
+    }
+
+    /**
+     * The method returns player name.
+     *
+     * @return String playerName
+     */
+    public String getPlayerName() {
```

Comment:
na ogol jak masz klase Player to wystarczy nazwac pole "name" -> raczej wiadomo, ze to name playera jak nie ma innego pola podobnego.